[2019-10-18 01:59:04,306 INFO] Loading train dataset from ../processed_data/Weibo_src50_conv100_tgt10_v50000.train.1.pt, number of examples: 32077
[2019-10-18 01:59:04,426 INFO]  * vocabulary size. source = 50004; target = 50004
[2019-10-18 01:59:04,426 INFO] Building model...
Traceback (most recent call last):
  File "../train.py", line 516, in <module>
    main()
  File "../train.py", line 500, in main
    model = build_model(model_opt, opt, fields, checkpoint)
  File "../train.py", line 382, in build_model
    use_gpu(opt), checkpoint)
  File "/Users/rishabh/Documents/Sem_3_Coursework/Information Extraction and Retrieval/Major Project/IREMajorProject/HashtagGeneration/onmt/ModelConstructor.py", line 225, in make_base_model
    model.cuda()
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 305, in cuda
    return self._apply(lambda t: t.cuda(device))
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 202, in _apply
    module._apply(fn)
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 202, in _apply
    module._apply(fn)
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 202, in _apply
    module._apply(fn)
  [Previous line repeated 1 more times]
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 224, in _apply
    param_applied = fn(param)
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 305, in <lambda>
    return self._apply(lambda t: t.cuda(device))
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/cuda/__init__.py", line 192, in _lazy_init
    _check_driver()
  File "/Applications/anaconda3/lib/python3.7/site-packages/torch/cuda/__init__.py", line 95, in _check_driver
    raise AssertionError("Torch not compiled with CUDA enabled")
AssertionError: Torch not compiled with CUDA enabled
